### ISSUE TITLE
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'types.GenericAlias'

### ISSUE THREAD

**Issue by DavidWalz** [m-wrzr/streamlit-searchbox#23]
Thank you for building this component!

I get an error when importing streamlit-searchbox 0.1.4 with python 3.9.1 due to the following type hint

```python
Traceback (most recent call last):
    from streamlit_searchbox import st_searchbox
  File "...\streamlit_searchbox\__init__.py", line 43, in <module>
    def _list_to_options_py(options: list[Any] | list[tuple[str, Any]]) -> list[Any]:
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'types.GenericAlias'
```

cf. https://bugs.python.org/issue42233

**Comment by m-wrzr** [comment]
Hey, thanks for opening the issue. 

Should be fixed in `streamlit-searchbox==0.1.5` - can you maybe confirm it's working again for your Python version?

**Comment by DavidWalz** [comment]
Works. Thank you for the super quick fix!
